app/models/token.rb

```diff
@@ -30,6 +30,7 @@ def public_key
     kid = decoded_token[1]['kid']
     key = OIDC::KeyService.get_key(kid)
     if key.blank?
+      StatsD.increment('okta_kid_lookup_failure', 1, ["kid:#{kid}"])
```

Comment:
Open to suggestions for other names for this metric. Also are there any other labels that would be useful here? The docs (https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/backend/sending-metrics-using-statsd.md#tagging) indicate that we won't be able to add additional dimensions to this metric in the future if we need to.

Reply:
I think this is fine. We aren't relying on this yet in any operational capacity. Once our debugging is complete and we've used the metric a bit w can rename it, either to add new labels or because we find a better name. New acronym: GEFN (good enough for now) :smile: